app/assets/javascripts/questions.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
This will be included in the pipeline, but not pass through `gulp`, which manages things in `app/gulp`.